spec/jekyll_seo_tag_spec.rb

```diff
@@ -247,6 +247,16 @@
           expect(output).to match(expected)
         end
       end
+
+      context 'with page.image' do
+        let(:site) { make_site('twitter' => site_twitter, 'url' => 'http://example.invalid') }
+        let(:page) { make_page('image' => 'foo.png') }
```

Comment:
Yes, it would. In my change for page image I copied the usage for site logo which prepends a / (https://github.com/jekyll/jekyll-seo-tag/blob/master/lib/template.html#L149) and documents it's use in that manner. I, personally, prefer starting the config value with a / and would be happy to make that change to both page image and site logo.